src/frontend/src/sass/_sideNav.scss

```diff
@@ -1,17 +1,19 @@
 .sidenav {
     height: 100%;
-    position: fixed;
     z-index: 1;
     background-color: transparent;
     overflow-x: hidden;
     transition: 0.5s;
-    width:$sidenav-width;
+    

```

Comment:
personally I would keep it fixed for a particular browser width.. So it does squash. Ideally this should be unde mediaqueries to make it responsive.